react/StoreName.tsx

```diff
@@ -1,19 +1,31 @@
-import React from 'react'
+import React, { FC } from 'react'
 import { useCssHandles } from 'vtex.css-handles'
 
 import { useStoreGroup } from './StoreGroup'
 
 const CSS_HANDLES = ['storeName'] as const
 
-const StoreName = () => {
+interface StoreNameProps {
+  text: string
+}
+
+const StoreName: FC<StoreNameProps> = ({ text }) => {
   const handles = useCssHandles(CSS_HANDLES)
   const group = useStoreGroup()
 
   if (!group) {
     return null
   }
 
-  return <div className={handles.storeName}>{group.friendlyName}</div>
+  const parseText = (string: string) => {
+    return string.replace(/{store-name}/gi, group.friendlyName)
+  }
+
+  return (
+    <h1 className={handles.storeName}>
```

Comment:
We should do this as a `Prop` because you can use this interface on any part of the page, not necessarily as an H1

Reply:
A good point. Will update this.